### ISSUE TITLE
On CentOS 7 Error executing action `start` on resource 'service[statsd]'

### ISSUE THREAD

**Issue by varunpalekar** [librato/statsd-cookbook#36]
It giving error on service start of `statsd`.

This happen due to not update the `systemd` service. So I have to manually do `systemctl daemon-reload` to reload it. Then after that only service `statsd` detects the new file changes. The full error is given below:

```
================================================================================
 Error executing action `start` on resource 'service[statsd]' ================================================================================
     Chef::Exceptions::Exec
     ----------------------
     /bin/systemctl start statsd returned 6, expected 0

     Resource Declaration:
     ---------------------
     # In /var/chef/cache/cookbooks/statsd/recipes/default.rb

     132: service 'statsd' do
     133:   provider chef_provider
     134:   supports restart: true, start: true, stop: true
     135:   action service_status
     136: end
     137:

     Compiled Resource:
     ------------------
     # Declared in /var/chef/cache/cookbooks/statsd/recipes/default.rb:132:in `from_file'

     service("statsd") do
       provider Chef::Provider::Service::Systemd
       action [:enable, :start]
       supports {:restart=>true, :start=>true, :stop=>true}
       retries 0
       retry_delay 2 
       guard_interpreter :default
       service_name "statsd"
       enabled true
       pattern "statsd"
       cookbook_name "statsd"
       recipe_name "default"
     end
```

**Comment by mheffner** [comment]
@stackexpress-varun Could you work up a patch for this? I don't believe anyone has tested/verified this on systemd machines yet.

**Comment by mheffner** [comment]
Fixed in #203
